packages/app/src/debug/DebugResults.vue

```diff
@@ -0,0 +1,67 @@
+<template>
+  <div
+    class="flex gap-8px items-center"
+    data-cy="debug-results-holder"
+  >
+    <ResultCounts
+      :total-passed="results.totalPassed"
+      :total-failed="results.totalFailed"
+      :total-skipped="results.totalSkipped"
+      :total-pending="results.totalPending"
+      :order="['PASSED', 'FAILED', 'SKIPPED', 'PENDING']"
+    >
+      <template #prefix>
```

Comment:
For information to show up on this page, does it have to fail? Is the `Failed` text always going to be visible?

Reply:
This is a good point! I just looked at the future milestones and looks like this text could be `Failed`, `Passed`, `Running`, e.t.c. So I will definitely change this to use a variable instead of a fixed value

Reply:
The "status" text will always be visible in the header component according to the designs